editor/src/components/tab_label/imp.rs

```diff
@@ -18,10 +18,12 @@ impl ObjectSubclass for TabLabel {
     type Type = super::TabLabel;
     type ParentType = gtk::Box;
 
+    #[inline]
     fn class_init(klass: &mut Self::Class) {
         Self::bind_template(klass);
     }
 
+    #[inline]
     fn instance_init(obj: &glib::subclass::InitializingObject<Self>) {
         obj.init_template();
     }
```

Comment:
Can you please remove `tab_label` completely? It's a dead code noww.

Reply:
Done https://github.com/null8626/Echidna/commit/c9122f2b8a39f73340328445788599dfc4c15be3